### ISSUE TITLE
Fail to generate install files for ISO with single image

### ISSUE THREAD

**Issue by knightcraw** [UXabre/WIMan#14]
GenerateWIM.ps1 fail to detect Windows name and silently skip the WIM generation when running against ISO with single image 
```
You cannot call a method on a null-valued expression.
At E:\WIMan\inc\DetectWindowsName.ps1:10 char:5
+     $windowsBase = $metaData.WIM.IMAGE[0].NAME.Substring(0, $metaData ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                     
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

**Comment by knightcraw** [comment]
$metaData.WIM.IMAGE is not an array for ISO with single image and $metaData.WIM.IMAGE[0] return $null

**Comment by knightcraw** [comment]
Fixed for my iso: https://github.com/UXabre/WIMan/pull/15

**Comment by UXabre** [comment]
Fixed by #15 , thanks for your contribution!
